addons/project/models/project.py

```diff
@@ -1859,8 +1825,21 @@ def _message_auto_subscribe_followers(self, updated_values, default_subtype_ids)
         return new_followers
 
     def _mail_track(self, tracked_fields, initial_values):
-        result = super()._mail_track(tracked_fields, initial_values)
-        changes, tracking_value_ids = result
+        changes, tracking_value_ids  = super()._mail_track(tracked_fields, initial_values)
+        # Many2many tracking
+        if len(changes) > len(tracking_value_ids):
+            for i, changed_field in enumerate(changes):
```

Comment:
`enumerate` is not necessary since we don't use the index. :) 
```suggestion
            for changed_field in changes:
```